src/components/Feedback/StyledInlineProgress.tsx

```diff
@@ -0,0 +1,57 @@
+import { Progress } from "antd";
+import Tooltip from "antd/es/tooltip";
+import React, { ReactElement, useContext, useRef } from "react";
+
+import { TOOLTIP_TRIGGER } from "../../constants";
+
+import { AppThemeContext } from "../AppStyle";
+
+type StyledInlineProgressProps = {
+  /** A `[0, 100]` integer percentage. */
+  percent: number;
+  error?: boolean;
+};
```

Comment:
I might've missed it, but is there a check to prevent `percent` from going negative?